timed/employment/filters.py

```diff
@@ -53,24 +52,18 @@ class Meta:
 class UserFilterSet(FilterSet):
     active      = NumberFilter(field_name='is_active')
     supervisor  = NumberFilter(field_name='supervisors')
-    is_reviewer = NumberFilter(method='filter_reviewers')
-    is_supervisor = NumberFilter(method='filter_supervisors')
+    is_reviewer = NumberFilter(method='filter_is_reviewers')
+    is_supervisor = NumberFilter(method='filter_is_supervisors')
 
-    def filter_reviewers(self, queryset, name, value):
-        reviewer = Project.objects.filter(reviewers=OuterRef('pk'))
-        return queryset.annotate(
-            is_reviewer=Exists(reviewer)
-        ).filter(is_reviewer=value)
+    def filter_is_reviewers(self, queryset, name, value):
+        return queryset.filter(pk__in=User.objects.all_reviewers())
```

Comment:
when value is 0 we should use exclude instead of filter to invert query. Same below.